add_task.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+todo_task_file=$1
+read -r -e -p "       ENTER NEW TASK: " task
+
+# Unselect the other tasks. Only the new task
+# should be selected.
+line_number=0
+lines=$( wc -l < "$todo_task_file" )
+while [[ $line_number -lt $((lines)) ]]; do
+  line_number=$((line_number+1))
+  task_line="$( sed "${line_number}q;d" "$todo_task_file" )"
+  task_text="$(echo $task_line| cut -d']' -f 2)"
```

Comment:
The line above has spaces in the start and end of the parentheses, seems odd to mix so close each other. :)